web/forms.py

```
@@ -1517,6 +1519,26 @@ class StudentEnrollmentForm(forms.Form):
     )
 
 
+import typing
```

Ruff (0.8.2)

Module level import not at top of file

(E402)